release_dashboard/lib/widgets/clean_release_button.dart

```diff
@@ -21,10 +21,12 @@ class CleanReleaseButton extends StatefulWidget {
   final ConductorService conductor;
 
   @override
-  State<CleanReleaseButton> createState() => _CleanReleaseState();
+  State<CleanReleaseButton> createState() => _CleanReleaseButtonState();
+
+  static const String requiredConfirmationString = 'clean release';
 }
 
-class _CleanReleaseState extends State<CleanReleaseButton> {
+class _CleanReleaseButtonState extends State<CleanReleaseButton> {
   String? _errorMsg;
 
   /// Updates [_errorMsg] with [errorMsg].
```

Comment:
Avoid referring to private variables in dartdocs

Reply:
Could you give an example of what should be the dartdocs here, please?